docker-for-mac/dashboard.md

```diff
@@ -16,7 +16,7 @@ The Docker Desktop Dashboard offers the following benefits:
 
 In addition, the Dashboard UI allows you to:
 
-- Navigate to the [Settings](/docker-for-mac/index/#preferences) menu to configure Docker Desktop preferences
+- Navigate to the [Settings](index.md#preferences) menu to configure Docker Desktop preferences
```

Comment:
The link doesn't work after the change. It works fine in prod now - https://docs.docker.com/docker-for-mac/dashboard/

Reply:
Hm.. that's weird; let me check

Reply:
Argh. it's the JavaScript thing that's messing up the link; Jekyll creates the correct link, but JavaScript breaks it 😞